features/cli/build.feature

```diff
@@ -753,6 +753,14 @@ Feature: build 'apps' with CLI
       | Build ConfigMaps:\s+cmtest->. |
       | Build Secrets:\s+secrettest->.|
     And the "ruby-hello-world-1" build completed
+    #Bug 1669981 comment #1, the secreti will not included in the output image.
+    When I run the :get client command with:
+      | resource      | po                       |
+      | resource_name | ruby-hello-world-1-build |
+      | o             | yaml                     |
+    Then the output should match:
```

Comment:
@xiuwang should this line be `Then the output should not match:` because of your comment in line 756?

Reply:
To adjust comment in line 756, here check secret in the build pod, not in the output image.